.vscode/cspell.json

```
@@ -32,6 +32,7 @@
     "sdk/applicationinsights/azure-applicationinsights/**",
     "sdk/batch/azure-batch/**",
     "sdk/ai/**/index/**",
+    "sdk/ai/azure-ai-generative/tests/synthetic_qa/**",
```

```suggestion
    "sdk/ai/azure-ai-generative/tests/**",
```

Let's make life easy for our other test devs and skip the entire tests dir